src/mono/wasm/runtime/library_mono.js

```diff
@@ -599,10 +599,23 @@ var MonoSupportLib = {
 			MONO.commands_received = buffer_obj;
 		},
 
+		mono_wasm_malloc_and_set_debug_buffer: function (command_parameters)
+		{
+			if (command_parameters.length > this._debugger_buffer_len)
+			{
+				if (this._debugger_buffer)
+					Module._free (this._debugger_buffer);
+				this._debugger_buffer_len = Math.max(command_parameters.length, this._debugger_buffer_len, 256);
```

Comment:
perhaps allocate 125%, so that possibly you don't have to re-allocate it next time. Otherwise approved :)

Reply:
I didn't see any message bigger than 150, so probably we will never re-allocate.